completions/completions_v2.py

```diff
@@ -184,10 +184,11 @@ def has_competions(self):
         return len(self._completions) > 0
 
     def handle_tabnine_commands(self, view, locations, prefix):
-        if len(self._results) == 1 and self._old_prefix is None:
+        if len(self._results) == 1 and self._old_prefix is None and prefix != "":
+            seperator = "::"
             existing = view.substr(sublime.Region(
-                max(locations[0] - (len(prefix) + 2), 0), locations[0]))
-            is_tabNine_command = existing == "::{}".format(prefix)
+                max(locations[0] - (len(prefix) + len(seperator)), 0), locations[0]))
```

Comment:
you can do the `"{}{}".format(seperator, prefix)` at line 188 and then just use `len(extracted_variable)`

Reply:
👍